# This file is distributed under the same license as the Django package.
#
# Translators:
# jamie lu <www.ljm969087551@qq.com>, 2017
# Jannis Leidel <jannis@leidel.info>, 2011
# Kevin Sze <leiarix@gmail.com>, 2012
# Lele Long <schemacs@gmail.com>, 2011,2015
# Liping Wang <lynn.config@gmail.com>, 2016-2017
# mozillazg <opensource.mozillazg@gmail.com>, 2016
# pylemon <leeway1985@gmail.com>, 2012-2013
# hizyn <zhangyanan5552@gmail.com>, 2016
# Ziang Song <songziang@gmail.com>, 2011
# Kevin Sze <leiarix@gmail.com>, 2012
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-04-24 19:23+0800\n"
"PO-Revision-Date: 2017-12-28 06:11+0000\n"
"Last-Translator: jamie lu <www.ljm969087551@qq.com>\n"
"Language-Team: Chinese (China) (http://www.transifex.com/django/django/"
"language/zh_CN/)\n"
"Language: zh_CN\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr "个人信息"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr "权限"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:49
msgid "Important dates"
msgstr "重要日期"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:135
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "具有主键 %(key)r 的对象 %(name)s 不存在。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:145
msgid "Password changed successfully."
msgstr "密码修改成功。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/admin.py:165
#, python-format
msgid "Change password: %s"
msgstr "修改密码：%s"

#: venv/lib/python3.6/site-packages/django/contrib/auth/apps.py:14
msgid "Authentication and Authorization"
msgstr "认证和授权"

#: venv/lib/python3.6/site-packages/django/contrib/auth/base_user.py:48
msgid "password"
msgstr "密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/base_user.py:49
msgid "last login"
msgstr "上次登录"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:30
msgid "No password set."
msgstr "密码未设置。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:35
msgid "Invalid password format or unknown hashing algorithm."
msgstr "不可用的密码格式或未知的哈希算法。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:70
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:307
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:379
msgid "The two password fields didn't match."
msgstr "两个密码字段不一致。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:73
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:126
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:163
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:383
msgid "Password"
msgstr "密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:79
msgid "Password confirmation"
msgstr "密码确认"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:82
#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:392
msgid "Enter the same password as before, for verification."
msgstr "为了校验，请输入与上面相同的密码。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:128
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"由于没有存储明文密码，所以无法查看该用户的密码，但是可以通过这个<a href="
"\"{}\">表单</a>修改密码。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:170
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr "请输入一个正确的 %(username)s 和密码. 注意他们都是大区分大小写的."

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:173
msgid "This account is inactive."
msgstr "该帐号未激活。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:234
msgid "Email"
msgstr "电子邮件"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:310
msgid "New password"
msgstr "新密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:316
msgid "New password confirmation"
msgstr "新密码确认"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:351
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "你的旧密码不正确。请重新输入。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:354
msgid "Old password"
msgstr "旧密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/forms.py:389
msgid "Password (again)"
msgstr "密码(重复)"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:263
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:337
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:434
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:496
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:527
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:563
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:599
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:637
msgid "algorithm"
msgstr "算法"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:264
msgid "iterations"
msgstr "迭代次数"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:265
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:343
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:436
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:497
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:528
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:638
msgid "salt"
msgstr "盐"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:266
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:344
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:498
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:529
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:564
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:600
#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:639
msgid "hash"
msgstr "哈希"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:338
msgid "variety"
msgstr "多样性"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:339
msgid "version"
msgstr "版本"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:340
msgid "memory cost"
msgstr "消耗内存"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:341
msgid "time cost"
msgstr "消耗时间"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:342
msgid "parallelism"
msgstr "对比"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:435
msgid "work factor"
msgstr "加密因子"

#: venv/lib/python3.6/site-packages/django/contrib/auth/hashers.py:437
msgid "checksum"
msgstr "校验和"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:56
#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:111
msgid "name"
msgstr "名称"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:60
msgid "content type"
msgstr "内容类型"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:62
msgid "codename"
msgstr "代码名称"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:66
msgid "permission"
msgstr "权限"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:67
#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:114
msgid "permissions"
msgstr "权限"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:121
msgid "group"
msgstr "组"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:122
#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:218
msgid "groups"
msgstr "组"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:209
msgid "superuser status"
msgstr "超级用户状态"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:212
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr "指明该用户缺省拥有所有权限。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:221
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr "该用户归属的组。一个用户将得到其归属的组的所有权限。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:229
msgid "user permissions"
msgstr "用户权限"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:231
msgid "Specific permissions for this user."
msgstr "为该用户声明权限。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:298
msgid "username"
msgstr "用户名"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:301
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr "必填。150个字符或者更少。包含字母，数字和仅有的@/./+/-/_符号。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:304
msgid "A user with that username already exists."
msgstr "已存在一位使用该名字的用户。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:307
msgid "first name"
msgstr "名字"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:308
msgid "last name"
msgstr "姓氏"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:309
msgid "email address"
msgstr "电子邮件地址"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:311
msgid "staff status"
msgstr "职员状态"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:313
msgid "Designates whether the user can log into this admin site."
msgstr "指明用户是否可以登录到这个管理站点。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:316
msgid "active"
msgstr "有效"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:319
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr "指明用户是否被认为活跃的。以反选代替删除帐号。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:323
msgid "date joined"
msgstr "加入日期"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:332
msgid "user"
msgstr "用户"

#: venv/lib/python3.6/site-packages/django/contrib/auth/models.py:333
msgid "users"
msgstr "用户"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] "密码长度太短。密码必须包含至少 %(min_length)d 个字符。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "你的密码必须包含至少 %(min_length)d 个字符。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "密码跟 %(verbose_name)s 太相似了。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can't be too similar to your other personal information."
msgstr "你的密码不能与其他个人信息太相似。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:185
msgid "This password is too common."
msgstr "这个密码太常见了。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:190
msgid "Your password can't be a commonly used password."
msgstr "你的密码不能是大家都爱用的常见密码。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:200
msgid "This password is entirely numeric."
msgstr "这个密码全部是数字的。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:205
msgid "Your password can't be entirely numeric."
msgstr "你的密码不能全部为数字。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "重置 %(site_name)s 的密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr "请输入合法的用户名。只能包含英文字母，数字和@/./+/-/_字符。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr "请输入合法用户名。只能包含字母，数字和@/./+/-/_ 字符。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:180
msgid "Logged out"
msgstr "退出登录"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:282
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:401
msgid "Password reset"
msgstr "密码重置"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:297
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:429
msgid "Password reset sent"
msgstr "密码重置链接已经发送。"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:335
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:438
msgid "Enter new password"
msgstr "输入新密码"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:346
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:498
msgid "Password reset unsuccessful"
msgstr "密码重置失败"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:366
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:506
msgid "Password reset complete"
msgstr "密码重置完成"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:541
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:569
msgid "Password change"
msgstr "密码更改"

#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:557
#: venv/lib/python3.6/site-packages/django/contrib/auth/views.py:592
msgid "Password change successful"
msgstr "密码更改成功"
